performance-tests/src/test/java/org/eclipse/collections/impl/parallel/SerialParallelLazyPerformanceTest.java

```diff
@@ -324,7 +324,7 @@ private MutableList<Function0<FastList<Integer>>> getIntegerListGenerators(int c
     private void measureAlgorithmForIntegerIterable(String algorithmName, Procedure<Function0<FastList<Integer>>> algorithm, boolean shuffle)
     {
         this.printMachineAndTestConfiguration(algorithmName);
-        for (int i = 0; i < 4; i++)
+        for (int i = 0; i < 1; i++)
```

Comment:
Are you shortening this test? Can you please add a big comment here of what is going on?

Reply:
Just added a comment here too.